### PR TITLE
More tau ceti foreign legion fixes

### LINKED ISSUE #5961: TCFL access issues
* TCFL have no access to air alarms and the engineers themselves must hack them open to access them.

* They also have incredibly limited medical access and often cannot interface with some machines.

### LINKED ISSUE #5964: TCFL don't spawn with bags mapped into the general TCFL base
This means they need bags adminbus'd in for them during events. They should have extra bags spawned in like ERT do.

### LINKED ISSUE #5965: TCFL have a chem dispenser at their base but not a ChemMaster
Title.

### PR DESCRIPTION
**Pull request by Alberyk**
-fixes #5961
-fixes #5965
-fixes #5964
-fixes the blaster carbine fitting inside holsters
-places the drop pods behind blast doors for now
-removes some non medical related chemicals from their dispenser

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
>-fixes the blaster carbine fitting inside holsters

Woops, was that my doing?

File: code/modules/projectiles/guns/energy/blaster.dm [hunk 1]
```diff
@@ -28,6 +28,7 @@
 	max_shots = 12
 	origin_tech = list(TECH_COMBAT = 3, TECH_MAGNET = 2)
 	projectile_type = /obj/item/projectile/energy/blaster
+	slot_flags = SLOT_BELT
```
Comment (Sindorman) [inline code/modules/projectiles/guns/energy/blaster.dm]: shouldn't this be `slot_flags = SLOT_BELT|SLOT_HOSLTER`?

Comment (OneOneThreeEight) [reply code/modules/projectiles/guns/energy/blaster.dm]: The slot flags should actually be `null` here. In-game energy carbines do not fit in any belt or holster.

**Comment by OneOneThreeEight** [comment]
`/obj/item/weapon/gun/energy/blaster` is a parent of `/obj/item/weapon/gun/energy/blaster/carbine` and `/obj/item/weapon/gun/energy/blaster/rifle`. If the latter two do not have written exceptions to certain variables, they will inherit the variables of their parent item. 

Take w_class, for instance. The blaster pistol as the parent item has w_class 3 by default, which is a normal sized item, same as a box. The blaster carbine doesn't have a written exception as opposed to its parent, and thus inherits the pistol's w_class and its slot_flags.

**Comment by Sindorman** [comment]
That's true, I did not see that.

**Pull request merged**